### PR TITLE
upgrade the app reuse file update

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to add note in the upgrade the app reuse file to redirect the user to the release note tab.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Add a note in upgrade app reuse template for v2 app to view changelog](https://sumologic.atlassian.net/browse/SUMO-242402)

### CODE REVIEW

**Review by jpipkin1 — APPROVED** [review]

File: docs/reuse/apps/app-update.md [hunk 1]
```diff
@@ -12,3 +12,7 @@ To update the app, do the following:
         1. Click **Next**. You will be redirected to the **Preview & Done** section.
 
 Your upgraded app will be installed in the **Installed Apps** folder and dashboard panels will start to fill automatically.
+
+:::note
+Go to the **Release Notes** tab to see the change log for new updates in the app.
```
Comment (kimsauce) [inline docs/reuse/apps/app-update.md:17]:
```suggestion
For individual app updates, see [**Service Release Notes**](/release-notes-service).
```

**Review by kimsauce — DISMISSED** [review]
had a suggestion

File: docs/reuse/apps/app-update.md (hunk 1 above)
Comment (jpipkin1) [reply docs/reuse/apps/app-update.md:17]: @kimsauce, the **Release Notes** tab appears in the App Catalog for apps that have an upgrade available:

Screenshot 2024-06-11 at 4 01 17 PM

Comment (kimsauce) [reply docs/reuse/apps/app-update.md:17]: Oh ok, thank you for clarifying! I don't think I ever noticed that tab was there.

**Pull request merged**